packages/db/scripts/cmds/db-sql.js

```diff
@@ -64,29 +45,56 @@ export async function dbSqlCmd ({ reset, cargo, testing } = {}) {
   await client.query(configSql)
   await client.query(tablesSql)
 
-  if (cargo) {
-    if (testing) {
-      cargoSql = cargoSql.replace(
-        `
--- Create materialized view from cargo "aggregate_entries" table
-CREATE MATERIALIZED VIEW public.aggregate_entry
-AS
-SELECT *
-FROM cargo.aggregate_entries;`,
-        `
-CREATE MATERIALIZED VIEW public.aggregate_entry
-AS
-SELECT *
-FROM cargo.aggregate_entries 
-WHERE cid_v1 in ('bafybeiaj5yqocsg5cxsuhtvclnh4ulmrgsmnfbhbrfxrc3u2kkh35mts4e');
-`
-      )
-    }
-
+  // if testing or cargo fdw flag not set, you just get the schema, no fdw connection to dagcargo
+  if (testing && cargo) {
+    await client.query(cargoSchema)
+  } else if (!testing && cargo) {
+    // Replace secrets in the FDW sql file
+    fdwSql = fdwSql.replace(":'DAG_CARGO_HOST'", `'${process.env.DAG_CARGO_HOST}'`)
+    fdwSql = fdwSql.replace(":'DAG_CARGO_DATABASE'", `'${process.env.DAG_CARGO_DATABASE}'`)
+    fdwSql = fdwSql.replaceAll(":'DAG_CARGO_USER'", `'${process.env.DAG_CARGO_USER}'`)
+    fdwSql = fdwSql.replaceAll(":'DAG_CARGO_PASSWORD'", `'${process.env.DAG_CARGO_PASSWORD}'`)
+    fdwSql = fdwSql.replaceAll(':WEB3_STORAGE_USER', `${process.env.WEB3_STORAGE_USER || 'CURRENT_USER'}`)
     await client.query(fdwSql)
     await client.query(cargoSql)
   }
 
   await client.query(functionsSql)
   await client.end()
 }
+
+const cargoSchema = `
+CREATE SCHEMA IF NOT EXISTS cargo;
+
+CREATE TABLE IF NOT EXISTS cargo.aggregate_entries (
+  aggregate_cid TEXT NOT NULL,
+  cid_v1 TEXT NOT NULL,
+  datamodel_selector TEXT NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS cargo.aggregates (
+  aggregate_cid TEXT NOT NULL UNIQUE,
+  piece_cid TEXT UNIQUE NOT NULL,
+  sha256hex TEXT NOT NULL,
+  payload_size BIGINT NOT NULL,
```

Comment:
```suggestion
  export_size BIGINT NOT NULL,
```

https://filecoinproject.slack.com/archives/C02BZPRS9HP/p1638980517480000